nmigen/back/verilog.py

```diff
@@ -65,6 +66,12 @@ def _convert_rtlil_text(rtlil_text, *, strip_internal_attrs=False, write_verilog
     if popen.returncode:
         raise YosysError(error.strip())
     else:
+        # Remove empty lines and things that are not legal Verilog
+        # syntax at start of file
+        verilog_text = "\n".join(itertools.dropwhile(
+            lambda x: not x or x.strip().startswith("--"),
```

Comment:
In general, avoid `not x` (and other implicit boolean conversions) in Python, since it obscures meaning by being less precise than necessary. A condition like `line == "" or line.startswith("--")` would be more clear here. (Is `strip` necessary? I thought Verific only uses `--` at the start of the line.)